test/helpers/fixture_sets/basic.rb

```diff
@@ -0,0 +1,63 @@
+module FixtureSetAssertions
+  class Basic < FixtureSet
+    def initialize(namespace)
+      @namespace = namespace
+      @app_name = "basic"
+    end
+
+    def assert_all_up
+      assert_unmanaged_pod_statuses("Succeeded")
+      assert_all_web_resources_up
+      assert_all_redis_resources_up
+      assert_configmap_data_up
+    end
+
+    def assert_unmanaged_pod_statuses(status, count=1)
+      pods = kubeclient.get_pods(namespace: namespace, label_selector: "type=unmanaged-pod,app=#{app_name}")
+      assert_equal count, pods.size, "Expected to find #{count} unmanaged pod(s), found #{pods.size}"
+      assert pods.all? { |pod| pod.status.phase == status }
+    end
+
+    def refute_managed_pod_exists
+      assert_unmanaged_pod_statuses("", 0)
```

Comment:
From the first look, it's not very clear what's the meaning of an empty string. I think it would way better readable to do something like `assert_equal 0, managed_pods.size`, since the purpose of this method is to make sure there are no managed pods